Foundation/FileHandle.swift

```diff
@@ -752,8 +752,10 @@ extension FileHandle {
     }
     
     internal static func _openFileDescriptorForURL(_ url : URL, flags: Int32, reading: Bool) throws -> Int32 {
```

Comment:
Again, I think its better to wrap this at the public API side where the call to `_openFileDescriptorURL` is made.

Reply:
The advantage of doing it here is you dont need to fix up every caller or any new callers.

Also the type for the file system representaion is not a `URL` or `String` so it would change the method signature.